src/android/app/build.gradle

```diff
@@ -70,4 +72,8 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation platform('com.google.firebase:firebase-bom:26.7.0')
+    implementation 'com.google.firebase:firebase-analytics'
```

Comment:
Why you use direbase-analytics, com.google.guava:listenablefuture?

Reply:
+